internal/collector/eth_get_balance_test.go

```diff
@@ -0,0 +1,49 @@
+package collector
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+)
+
+func TestEthGetBalance(t *testing.T) {
+	rpcServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		_, err := w.Write([]byte(`{"result": "0x17b6d1ef1dff6b88"}"`))
+		if err != nil {
+			t.Fatalf("could not write a response: %#v", err)
+		}
+	}))
+	defer rpcServer.Close()
+
+	rpc, err := rpc.DialHTTP(rpcServer.URL)
+	if err != nil {
+		t.Fatalf("rpc connection error: %#v", err)
+	}
+
+	collector := NewEthGetBalance(rpc, "0x7A6A59588B8106045303E1923227a2cefbEC2B66")
```

Comment:
Why that specific address?

Reply:
Nothing specific, changed to a more random/mocked address.